pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -153,6 +153,9 @@ public CmdTopics(PulsarAdmin admin) {
         jcommander.addCommand("get-inactive-topic-policies", new GetInactiveTopicPolicies());
         jcommander.addCommand("set-inactive-topic-policies", new SetInactiveTopicPolicies());
         jcommander.addCommand("remove-inactive-topic-policies", new RemoveInactiveTopicPolicies());
+        jcommander.addCommand("get-maxConsumers", new GetMaxConsumers());
+        jcommander.addCommand("set-maxConsumers", new SetMaxConsumers());
+        jcommander.addCommand("remove-maxConsumers", new RemoveMaxConsumers());
```

Comment:
```suggestion
        jcommander.addCommand("get-max-consumers", new GetMaxConsumers());
        jcommander.addCommand("set-max-consumers", new SetMaxConsumers());
        jcommander.addCommand("remove-max-consumers", new RemoveMaxConsumers());
```

Reply:
And please also update the max producers as the above pattern.

Reply:
Done.